README.md

```diff
@@ -1,2 +1,155 @@
-# Hello-Microverse
-This simple project shows how to set up a "Hello Microverse" repository to demonstrate Linter configuration and GitHub action.
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello-Microverse] <a name="about-project"></a>
+
+**[Hello-Microverse]** is a simple project shows how to set up a "Hello Microverse" repository to demonstrate Linter configuration and GitHub flow.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> This simple project bulit with HTML and CSS
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The following are the key features of the application.
+
+- **[Display only Hello Microverse with crimson color]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Click the live demo to access the application.
+
+- [Live Demo Link](https://mulugithub.github.io/Hello-Microverse/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Describe how a new developer could make use of your project.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+### Setup
+
+Clone this repository to your desired folder:
+git clone git@github.com:mulugithub/Hello-Microverse.git
+
+
+### Install
+
+Install this project with:
+npm init -y 
+npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+
+### Run tests
+
+To run tests, run the following command:
```

Comment:
Kindly avoid incomplete statements in your readme as they give it a less professional look and make it less easy for other users😉. Kindly add the right command required to use this project👍.

since there's no test in this project you can take the section away or add N/A indicating that there's no test currently.